LIPs/LIP-treasury_contribution_percentage.md

```diff
@@ -0,0 +1,92 @@
+---
+lip: treasury_contribution_percentage
+title: Treasury Contribution Percentage
+author: Doug Petkanics (@dob), Victor Elias (@victorges)
+type: Standard Track
+status: Draft
+created: 2023-06-14
+discussions-to: https://forum.livepeer.org/t/livepeer-delta-phase-pre-proposal-sustainability-public-goods-funding-treasury-and-decentralization/2056
```

Comment:
Recommend separating out the discussion thread for this LIP vs. the treasury bundle so that there are dedicated areas for each topic instead of lumping everything together into a single discussion thread.